### PR TITLE
Add documentation for label selector

### PR DESCRIPTION
**Pull request by johscheuer**
This PR add documentation for the `label_selector` feature discussed here: https://github.com/kiali/kiali/issues/1214

Fixes: https://github.com/kiali/kiali/issues/1214

### CODE REVIEW

File: content/documentation/getting-started/index.adoc [hunk 1]
```diff
@@ -367,6 +367,17 @@ namespaces:
     - mycorp_test.*
 ----
 
+==== Namespace Selectors
+
+Kiali supports an optional label selector for namespaces which is used to fetch a subset of the available namespaces. The label selector is defined under the namespaces definition. The example below selects all namespaces that have a label `kiali-enabled: true`.
```
Comment (pilhuhn) [inline content/documentation/getting-started/index.adoc]: ( While not the general style of the file, it is a good idea to start a new sentence on a new line  -- no need to change it now )

Would it make sense to show the command to label a namespace with that selector for people that are less familiar? Or link to the kube documentation?

**Comment by pilhuhn** [comment]
Thank you for writing this up!

File: content/documentation/getting-started/index.adoc [hunk 2]
```diff
@@ -367,6 +367,17 @@ namespaces:
...
+
+[source,yaml]
+----
+namespaces:
+  label_selector: kiali-enabled=true
+----
+
```
Comment (jmazzitelli) [inline content/documentation/getting-started/index.adoc]: I would want to talk about the reason why we originally added this label_selector - and that is to support soft-multitenancy. Particularly, we need to tell people the value of that label selector should be unique across multiple meshes. My suggested addition is below:

```suggestion

Note that when using multiple service meshes (i.e. multiple control planes) in the same cluster, you will want to set the label selector's value to a value unique to each mesh. This is so each mesh's Kiali instance will only select those namespaces within that mesh. 
For an example of using Kiali in this kind of soft multi-tenancy mode, see the [Maistra](https://github.com/Maistra/istio-operator) project.
This is the reason why this `label_selector` will be defined by default to the value of `kiali.io/member-of: <istio_namespace>` if the `deployment.accessible_namespaces` is set to something other than the "all namespaces" value `['**']`. This allows you to have multiple control planes in the same cluster and have each control plane contain its own Kiali instance.

```

**Comment by jmazzitelli** [comment]
FYI: @jshaughn will be unable to review for a couple weeks.

File: content/documentation/getting-started/index.adoc (hunk 1 above)
Comment (johscheuer) [reply content/documentation/getting-started/index.adoc]: I added it

File: content/documentation/getting-started/index.adoc (hunk 2 above)
Comment (johscheuer) [reply content/documentation/getting-started/index.adoc]: Updated the docs

File: content/documentation/getting-started/index.adoc [hunk 3]
```diff
@@ -367,6 +367,38 @@ namespaces:
...
+The example below selects all namespaces that have a label `kiali-enabled: true`:
+
+[source,yaml]
+----
+namespaces:
+  label_selector: kiali-enabled=true
+----
+
+For further information how the `label_selector` interacts with `deployment.accessible_namespaces` read the https://github.com/kiali/kiali/blob/v1.1.0/operator/deploy/kiali/kiali_cr.yaml#L44-L53[technical documentation].
```
Comment (jshaughn) [inline content/documentation/getting-started/index.adoc]: minor:   "information how" => " information on how"

Also, I'd avoid versioning and line numbers in the link.  At the moment we don't have a versioning solution in the docs (we need that).  And line number can be brittle.  How about:

read the 'label_selector' documentation in https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[kiali_cr.yaml].

**Comment by israel-hdez** [comment]
I'm merging.
Thanks @johscheuer for your contribution.

**Pull request merged**